.github/workflows/apidoc.yaml

```diff
@@ -23,7 +23,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        apiGroup: [ 'public-api', 'management-api', 'control-api' ]
+        apiGroup: [ 'default-api', 'public-api', 'management-api', 'control-api' ]
```

Comment:
@paullatzelsperger  do we really want to publish these in Swagger hub?

Reply:
well, we need to document it _somewhere_, so why not Swaggerhub.